lib/turn.rb

```diff
@@ -0,0 +1,26 @@
+class Turn
+    attr_reader :guess, :card #these will recieve input; if only output do not include
+    def initialize(guess, card)#arguments 
+        @guess = guess
+        @card = card
+    end
+    
+    def correct?
+        @guess == card.answer
```

Comment:
In this method, you're calling to card instead of @card. This works, because you're calling to the attr_reader on line 2, which returns the instance variable on line 5. Since you have access to the instance variable, call to that rather than the attr_reader. The purpose of the attr reader is so that you can call to that attribute when you're outside of this class.